Helper/Config.php

```diff
@@ -434,12 +435,18 @@ public function debugLogsEnabled($storeId): bool
 
     public function getAutoCaptureOpenInvoice(int $storeId): bool
     {
-        return $this->getConfigData('auto_capture_openinvoice', self::XML_ADYEN_ABSTRACT_PREFIX, $storeId, true);
+        $captureForOpenInvoice = $this->getConfigData('capture_for_openinvoice', self::XML_ADYEN_ABSTRACT_PREFIX, $storeId, true);
+        if (strcmp((string) $captureForOpenInvoice, self::AUTO_CAPTURE_OPENINVOICE) === 0) {
+            return true;
+        }
+        else {
+            return false;
+        }
```

Comment:
```suggestion
        return strcmp((string) $captureForOpenInvoice, self::AUTO_CAPTURE_OPENINVOICE) === 0
```

Reply:
As per my findings strcmp returns the value. 
ref: https://www.php.net/manual/en/function.strcmp.php

Reply:
My comment is related to the `if` statement, you can remove it as my code suggestion stated